packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -8,30 +8,43 @@ import type { ProxyConfiguration } from './ProxyConfiguration';
 import type { SdkVerbosity } from './SdkVerbosity';
 import { TrackingConsent } from './TrackingConsent';
 
+const DEFAULTS = {
```

Comment:
is there any way to make the content of this object immutable? I guess I can do something like `DEFAULTS. resourceTracingSamplingRate = 100` now?

Reply:
yes we can use `Object.freeze`. However I don't really see a case where this would happen? `DEFAULTS` is not available to SDK users

Reply:
I see. It was just my curiosity regarding the language syntax.